docs/sources/mimir/manage/tools/query-tee.md

```diff
@@ -89,6 +89,14 @@ A request sent from the query-tee to a backend includes HTTP basic authenticatio
 - If the backend endpoint URL is configured only with a username, then query-tee keeps the configured username and injects the password received in the incoming request.
 - If the backend endpoint URL is configured without a username and password, then query-tee forwards the authentication credentials found in the incoming request.
 
+### Backend selection
+
+The query-tee supports always sending requests to all backends, or sending only a proportion of requests to all backends, with the remainder sent only to the preferred backend.
+This is configured with `-proxy.secondary-backends-request-proportion`.
```

Comment:
```suggestion
You can use the query-tee to either send requests to all backends or to send a proportion of requests to all backends and the remaining requests to your preferred backend.
Configure backend selection in the `-proxy.secondary-backends-request-proportion` setting.
```

Reply:
Thank you for making these updates! I left a really minor comment, but these changes look great to me!